src/Solr/Reindex/Handlers/SolrReindexImmediateHandler.php

```diff
@@ -97,6 +97,13 @@ protected function processGroup(
 
         // Execute script via shell
         $process = new Process($cmd);
+
+        // Set timeout from config. Process default is 60 seconds.
+        $timeout = Config::inst()->get(static::class, 'process_timeout');
+        if ($timeout) {
```

Comment:
Yeah, as @NightJar said, null is a valid value to pass through. This if statement could be removed entirely